sdk/search/Azure.Search.Documents/tests/DocumentOperations/VectorSearchTests.cs

```diff
@@ -92,5 +92,48 @@ public async Task SimpleHybridSearch()
                 h => h.Document.HotelId,
                 "3", "1", "2", "10", "4", "5", "9");
         }
+
+        [Test]
+        public async Task SemanticHybridSearch()
+        {
```

Comment:
I I think the live test in nightly pipeline will fail for this because today we use basic tier to create search resources. In order to run the live test for semantic search, it requires following as per https://learn.microsoft.com/en-us/azure/search/semantic-search-overview#enable-semantic-search:

- Minimum standard tier for Search Service-- We have to update ⁠[basic](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/test-resources.json#L60) to standard
- Select a semantic plan Free/Standard --- We can use free but I don't know how to select a plan using our test resources set up in the pipelines.

@pallavit should we update ⁠[basic](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/test-resources.json#L60) to standard and I'll see how to select and set a semantic plan in the pipeline or should we run the semantic search test in Playback mode only?

Reply:
I think we should move to standard. To check what change is needed in the Bicep file. If you create the resource in portal with the semantic plan - you can see ARM settings - by downloading the ARM template. Also, I am sure Beverly will be able to help us with it as well :)